susemanager-utils/susemanager-sls/src/modules/kiwi_info.py

```diff
@@ -37,6 +37,22 @@ def parse_buildinfo(dest):
                 group[match.group('name')] = match.group('val')
     return ret
 
+def guess_buildinfo(dest):
```

Comment:
Is this related to SLE11?

Reply:
Yes, SLES11 Kiwi does not create the buildinfo file, so the information has to be extracted this way.